### PR TITLE
Add discounts to invoices draft and update

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

**Review by jenssegers — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -1445,11 +1445,11 @@ Get details for a single invoice.
                         + id: `f88ab83c-b8ca-497b-a2c6-1ad631fe1cea` (string)
             + discounts (array)
                 + (object)
-                    + description: `Discount` (string)
                     + type: `percentage` (enum[string])
                         + Members
                             + percentage
                     + value: 15.00 (number)
+                    + discount: `Discount` (string)
```
Comment (WouterSioen) [inline apiary.apib]: Q: Shouldn't this be description?

Comment (luisaterrabrito) [reply apiary.apib]: yes, thanks :)

 :ok: by @luisabritotl on 26/03/2018 15:27:20

**Review by WouterSioen — APPROVED** [review]

File: src/05-invoicing/invoices.apib [hunk 2]
```diff
@@ -184,6 +184,13 @@ Draft a new invoice.
                                     + excluding
                         + tax_rate_id: `c0c03f1e-77e3-402c-a713-30ea1c585823` (string, required)
         + invoice_date: `2016-02-04` (string, optional)
+        + discounts (array)
+            (object)
+                + value: 0.5123 (number, required)
+                + type (enum[string], required)
+                    + Members
+                        + percentage
+                + description: `winter promotion` (string) - If the description is not given, the proper translation of the word discount will be used in the user interface
```
Comment (jdrieghe) [inline src/05-invoicing/invoices.apib:193]: Let's do it in multiple phases and leave that comment out for now (or make the string 'optional', but I think it's better to take smaller steps)

**Review by jdrieghe — CHANGES_REQUESTED** [review]
let's go for an even smaller step

**Comment by luisaterrabrito** [comment]
Opening smaller PRs

**Pull request closed without merging**